class FailureMessages {
  FailureMessages._();

  static const unexpectedFailure =
      'Something went wrong. Please try again later.';
}
